From f1c872817b2787f18f0a8f7019274f1be49e2a4c Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Tue, 1 Jun 2010 10:27:39 +0900
Subject: [PATCH] Specification of date format -- bug #569174.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RFC 822 dates use only two digits for the years, but Debian changelogs
described by this paragraph (ยง4.4 in Policy 3.8.4) use four digits.

This patch replaces the reference to the RFC 822 by a specification that is
compatible with its successors, RFC 2822 and RFC 5322, but does not use their
full range of options.
---
 policy.sgml |   26 ++++++++++++++++++++++----
 1 files changed, 22 insertions(+), 4 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index af00c0e..5ba1980 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -1618,11 +1618,29 @@
 	</p>
 
 	<p>
-	  The <var>date</var> must be in RFC822 format<footnote>
+	  The <var>date</var> has the following format<footnote>
 	      This is generated by <tt>date -R</tt>.
-	  </footnote>; it must include the time zone specified
-	  numerically, with the time zone name or abbreviation
-	  optionally present as a comment in parentheses.
+	  </footnote> (compatible and with the same semantics of
+	  RFC 2822 and RFC 5322):
+	  <example>day-of-week, dd month yyyy hh:mm:ss +zzzz</example>
+	  where: 
+	  <list compact="compact">
+	    <item>day-of week is one of: Mon, Tue, Wed, Thu, Fri, Sat, Sun</item>
+	    <item>dd is a one- or two-digit day of the month (01-31)</item>
+	    <item>month is one of: Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct, Nov, Dec</item>
+	    <item>yyyy is the four-digit year (e.g. 2010)</item>
+	    <item>hh is the two-digit hour (00-23</item>
+	    <item>mm is the two-digit minutes (00-59)</item>
+	    <item>ss is the two-digit seconds (00-60)</item>
+	    <item>
+	      +zzzz or -zzzz is the the time zone offset from Coordinated Universal
+	      Time (UTC).  "+" indicates that the time is ahead of (i.e., east of) UTC
+	      and "-" indicates that the time is behind (i.e., west of) UTC.  The
+	      first two digits indicate the hour difference from UTC and the last
+	      two digits indicate the number of additional minutes difference from
+	      UTC.  The last two digits must be in the range 00-59.
+	    </item>
+	  </list>
 	</p>
 
 	<p>
-- 
1.6.5.7

